proto/protocolapis/astria/protocol/asset/v1alpha1/types.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package astria.protocol.asset.v1alpha1;
+
+// A response containing the denomination given an asset ID.
+message DenomResponse {
+  uint64 height = 2;
```

Comment:
should this start at 1?

Reply:
Yeah, they should all start at 1. Too late now because that would be a breaking change. :shrug: